day_6/exercises/burrito.rb

```diff
@@ -5,15 +5,32 @@
 # 3. change_protein
 
 class Burrito
-  attr_reader :protein, :base, :toppings
+  attr_accessor :protein, :base, :toppings
   def initialize(protein, base, toppings)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(t)
+    self.toppings.push(t)
+  end
+
+  def remove_topping(t)
```

Comment:
Interesting implementation here! Sometimes - whatever works is whats best. But! ruby has a built in delete method as well that reads a little better. Good job on this exercise which combines some core Mod-1 skills.